docs/examples/zoom.ipynb

```diff
@@ -0,0 +1,285 @@
+    "from esssans.types import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64783c51",
+   "metadata": {},
+   "source": [
+    "### Setup input files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9e7cfc82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params = {\n",
+    "    sans.types.DirectBeamFilename: 'Direct_Zoom_4m_8mm_100522.txt',\n",
+    "    sans.isis.CalibrationFilename: '192tubeCalibration_11-02-2019_r5_10lines.nxs',\n",
+    "    sans.isis.Filename[sans.types.SampleRun]: 'ZOOM00034786.nxs',\n",
+    "    sans.isis.Filename[sans.types.EmptyBeamRun]: 'ZOOM00034787.nxs',\n",
+    "}\n",
+    "masks = [\n",
+    "    'andru_test.xml',\n",
+    "    'left_beg_18_2.xml',\n",
+    "    'right_beg_18_2.xml',\n",
+    "    'small_bs_232.xml',\n",
+    "    'small_BS_31032023.xml',\n",
+    "    'tube_1120_bottom.xml',\n",
+    "    'tubes_beg_18_2.xml',\n",
+    "]\n",
+    "masks = sciline.ParamTable(str, {sans.isis.PixelMaskFilename: masks}, index=masks)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a8a4e01",
+   "metadata": {},
+   "source": [
+    "### Setup reduction parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50f11eb2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params[NeXusMonitorName[Incident]] = 'monitor3'\n",
+    "params[NeXusMonitorName[Transmission]] = 'monitor5'\n",
+    "\n",
+    "band = sc.linspace('wavelength', 1.75, 16.5, num=2, unit='angstrom')\n",
+    "params[WavelengthBands] = band\n",
+    "params[WavelengthBins] = sc.geomspace(\n",
+    "    'wavelength', start=band[0], stop=band[-1], num=141\n",
+    ")\n",
+    "\n",
+    "params[QBins] = sc.geomspace(dim='Q', start=0.004, stop=0.8, num=141, unit='1/angstrom')\n",
+    "params[NonBackgroundWavelengthRange] = sc.array(\n",
+    "    dims=['wavelength'], values=[0.7, 17.1], unit='angstrom'\n",
+    ")\n",
+    "params[CorrectForGravity] = True\n",
+    "params[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.upper_bound"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b47024f",
+   "metadata": {},
+   "source": [
+    "### Setup reduction pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fdcaec37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "providers = sans.providers + sans.isis.providers\n",
+    "providers = providers + (\n",
+    "    sans.isis.transmission_from_background_run,\n",
+    "    sans.isis.transmission_from_sample_run,\n",
+    ")\n",
+    "pipeline = sciline.Pipeline(providers, params=params)\n",
+    "pipeline.set_param_table(masks)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "951aec10",
+   "metadata": {},
+   "source": [
+    "If Mantid is available, we can use it to load data files.\n",
+    "**You must configure the `DataFolder` below to point to the directory containing the data files.**\n",
```

Comment:
`DataFolder` renders strangely in the docs
![Screenshot at 2024-01-18 14-15-30](https://github.com/scipp/esssans/assets/39047984/95bf8b0a-aca5-4a7c-bb06-95f98073ea8d)